src/sql_utils.h

```diff
@@ -9,13 +9,21 @@
 
 // Unique ownership SQLite3 statement creation
 template <typename T>
-static void bindArguments(sqlite3* db, sqlite3_stmt* statement, int cnt, int v) {
+static void bindArgument(sqlite3* db, sqlite3_stmt* statement, int cnt, int v) {
```

Comment:
@lbonn Can you ACK this change, please?

Reply:
Yes that was a typo... Nice!

It wouldn't surface before because in all our bindings, the `int` values where the last in the type list (the only case that was covered, accidentally)

Reply:
Because it was a typo. Without this change we cannot bind the integer arguments